### PR TITLE
Update docblocks

### PR DESCRIPTION
**Pull request by nunorafaelrocha**

### CODE REVIEW

File: lib/Bitreserve/BitreserveClient.php [hunk 1]
```diff
@@ -155,7 +155,9 @@ public function getTicker()
     /**
      * Retrieve all exchanges rates relative to a given currency.
      *
-     * @return array The current rates for the currency specific as provided by the ticker.
+     * @param  string $currency The filter currency.
```
Comment (nunofgs) [inline lib/Bitreserve/BitreserveClient.php]: Please remove all of the extra whitespaces after @param.

**Comment by nunorafaelrocha** [comment]
@nunofgs and @ruimarinho I've just finish modifying all aliments and extra whitespace. :smile:

**Comment by nunofgs** [comment]
@nunorafaelrocha LGTM. Is there a setting we can enable on our dotfiles so that DocBlockr doesn't add the extra whitespaces automatically?

**Comment by nunorafaelrocha** [comment]
Our `dotfiles` already have that configuration in Sublime Preferences. It's the `"jsdocs_align_tags": "no"` :smile:

**Pull request merged**